apps/template/src/App.tsx

```diff
@@ -55,6 +55,7 @@ export const App = (props: AppProps) => {
         embedded={props.embedded === "true" || props.embedded === true}
         sideNavigation={null}
         topNavigation={null}
+        fullWidthContent={false}
```

Comment:
Was this setting intentional, or was it just a test? I was under the impression that our default app setup behavior should be:  
- **embedded = true** → content spans the full width.  
- **embedded = false** → content  in width-constrained, centred.  

By explicitly setting `fullWidthContent` to `false` in the template app, the content will always be constrained to a centered column with a limited width, regardless of the value of `embedded`.

Reply:
Yes, true. Good catch. I didn't want to specify this prop at all in the template app but got a typecheck error when I left it out (see above). I guess what we should do instead is the same as for the `embedded` prop, i.e. use the value that was passed from outside if available, undefined otherwise.